science/seqan3/Portfile

```diff
@@ -1,36 +1,35 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem            1.0
-PortGroup             cmake 1.1
-PortGroup             compilers 1.0
-PortGroup             github 1.0
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               compilers 1.0
+PortGroup               github 1.0
 
-github.setup          seqan seqan3 3.1.0
-checksums             rmd160  ba4b16bbd777f148e53bd74a78f3e3c7a6a3de27 \
-                      sha256  0b37b1c3450e19c0ebe42c052c3f87babb8074bd772f10a553949c312c285726 \
-                      size    2656120
+github.setup            seqan seqan3 3.2.0
+checksums               rmd160  5525c4fcabc8eda171279b6d5632b37b6256ae6c \
+                        sha256  2737a9dc20c1d728674010662c0c9452aabbb3d4a64725bda24f28335468994c \
+                        size    2339440
 
-categories            science
-supported_archs       noarch
-license               BSD
-maintainers           {fu-berlin.de:rene.rahn rrahn} openmaintainer
-description           SeqAn3 - The modern C++ library for sequence analysis
-long_description      SeqAn3 is the new version of the popular SeqAn template \
-                      library for the analysis of biological sequences. \
-                      It enables the rapid development of high-performance solutions \
-                      by providing generic algorithms and data structures \
-                      By leveraging modern C++ it provides unprecedented ease-of-use \
-                      without sacrificing performance.
-platforms             darwin
-homepage              https://www.seqan.de
+categories              science
+supported_archs         noarch
+license                 BSD
+maintainers             {fu-berlin.de:rene.rahn rrahn} openmaintainer
```

Comment:
please add the `@` for the GitHub handle so that the maintainer will be notified automatically

Reply:
> please add the `@` for the GitHub handle so that the maintainer will be notified automatically

@reneeotten Thanks, will do it!